### PR TITLE
import-config backs up your ~/.ssh/config

### PR DESCRIPTION
**Pull request by edulop91**

### CODE REVIEW

**Comment by edulop91** [comment]
we already prompt the user to overwrite/append/nothing to their ssh config. not sure if this is too much

**Comment by edulop91** [comment]
@tfrcarvalho, what do you think?

File: cmd/import-config.go [hunk 1]
```diff
@@ -138,6 +139,14 @@ func sshConfig(conf *config.Config) error {
 		return nil // nothing to do
 	}
 
+	sshDir := path.Dir(conf.ClientConfig.SSHPrivateKey)
+	sshConfPath := path.Join(sshDir, "config")
+	err := backupFile(sshConfPath, fmt.Sprintf("%s.bak", sshConfPath))
```
Comment (ryanking) [inline cmd/import-config.go]: this only works if there isn't already a .bak. To be really safe we probably need bak1, bak2, etc.

Comment (edulop91) [reply cmd/import-config.go]: put a timestamp instead

**Comment by tfrcarvalho** [comment]
SGTM

**Pull request merged**